src/js/components/hyperdrive-size/index.js

```diff
@@ -30,8 +30,10 @@ HyperdriveSize.prototype._updateSize = function (state) {
     var s = state.archiveReducer
     if (s.archive && s.archive.content && s.archive.content.bytes) {
       this._size = prettyBytes(s.archive.content.bytes)
+      this.$el.style.display = 'block'
     } else {
       this._size = 0
+      this.$el.style.display = 'none'
```

Comment:
the reducer in the size component wasn't run for the downloading client, and it'll be 0 all the time and weird.  i suggest we hide that for now and rework size styles along with the reducer fixes.